src/main/java/data/plans/PlanList.java

```diff
@@ -72,7 +72,7 @@ public Plan createAndAddPlan(String userArgument) throws ArrayIndexOutOfBoundsEx
 
         ArrayList<Workout> workoutsToAddInAPlanList = new ArrayList<Workout>();
         for (int i = 0; i < numberOfWorkoutsInAPlan; i += 1) {
-            int workoutNumberInteger = Integer.parseInt(userWorkoutNumbersString.split(",")[i]);
+            int workoutNumberInteger = Integer.parseInt(userWorkoutNumbersString.split(",")[i].trim());
```

Comment:
Effective and accurate detection of bug!

Reply:
props to Alan ^_^!